rust/cubesql/cubesql/src/sql/session.rs

```diff
@@ -62,6 +62,8 @@ pub enum QueryState {
 pub struct SessionState {
     // connection id, immutable
     pub connection_id: u32,
+    // Can be UUID or anything else. MDX uses UUID
+    pub extra_id: Option<String>,
```

Comment:
Do we want to limit `extra_id`  to UTF-8? Maybe `Vec<u8>`/`Bytes` would be better. UUID is representable with 16 raw bytes, AFAIK `uuid` crate does exactly that.

Reply:
👍